obelisk/cpp/utils/include/msg_conversions.h

```diff
@@ -0,0 +1,146 @@
+#include <unsupported/Eigen/CXX11/Tensor>
+
+#include "obelisk_std_msgs/msg/float_multi_array.hpp"
+#include "obelisk_std_msgs/msg/u_int8_multi_array.hpp"
+
+namespace obelisk::utils::msgs {
+    namespace internal {
+        /**
+         * @brief Internal helper function to create tensors of arbitrary dimension.
+         *
+         * @param data the data to assign to the tensor
+         * @param dims the dimensions of the tensor
+         * @param Indices the index sequence to use for the parameter pack
```

Comment:
why uppercase?

Reply:
It's technically a template argument, so i've named it as such